changelog.d/1937.breaking.rst

```diff
@@ -0,0 +1 @@
+Preserved case-sensitivity of keys in setup.cfg so that entry point names are case-sensitive. Changed sensitivity of configparser -- by :user:`melissa-kun-li`
```

Comment:
I'm on the fence for declaring this backward-incompatible, though strictly it is. Often when making a backward-incompatible change, it's important to let the users know what action they should expect to take to account for the incompatibility. This message implies what to do and it's good enough. You might consider something like, "Any projects relying on the case-insensitivity will need to adapt to accept the original case as published."

Reply:
Hmmm, I thought about it more and I think I'm going to put it in the category of `change` instead of `breaking`. I think it would be a compatible change with the vast majority of users as it still maintains the convention of lower-case, and in addition is also a bug-fixing change for many other users who require meaningful case-sensitivity. For a smaller subsection of users it may be breaking, but overall I think it could be considered backwards-compatible. I'll also add that message you mentioned so that it's clear!

Reply:
Sounds good, but it looks like this file is still present. I'll remove it.

Reply:
```
setuptools main $ hub pr checkout 2580
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (22/22), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 22 (delta 10), reused 22 (delta 10), pack-reused 0
Unpacking objects: 100% (22/22), 11.48 KiB | 167.00 KiB/s, done.
From https://github.com/pypa/setuptools
 * [new ref]           refs/pull/2580/head -> fix-entry-points-name-case-sensitivity
Switched to branch 'fix-entry-points-name-case-sensitivity'
setuptools fix-entry-points-name-case-sensitivity $ git rm changelog.d/1937.breaking.rst
rm 'changelog.d/1937.breaking.rst'
setuptools fix-entry-points-name-case-sensitivity $ git commit -a -m "Remove 'breaking' changelog, superseded by 'change'."
 
[fix-entry-points-name-case-sensitivity 898a0b59] Remove 'breaking' changelog, superseded by 'change'.
 1 file changed, 1 deletion(-)
 delete mode 100644 changelog.d/1937.breaking.rst
setuptools fix-entry-points-name-case-sensitivity $ git push
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 4 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 322 bytes | 161.00 KiB/s, done.
Total 3 (delta 2), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To github.com:melissa-kun-li/setuptools.git
   11529db0..898a0b59  fix-entry-points-name-case-sensitivity -> fix-entry-points-name-case-sensitivity
```